solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/impl/TestValueFeature.java

```diff
@@ -47,15 +48,29 @@ public static void after() throws Exception {
     aftertest();
   }
 
-  @Test(expected = Exception.class)
+  @Test
   public void testValueFeatureWithEmptyValue() throws Exception {
-    loadFeature("c2", ValueFeature.class.getCanonicalName(), "{\"value\":\"\"}");
+    final RuntimeException expectedException = 
+        new RuntimeException("mismatch: '0'!='400' @ responseHeader/status");
+    try {
+        loadFeature("c2", ValueFeature.class.getCanonicalName(), "{\"value\":\"\"}");
+        fail("unexpectedly got here instead of catching "+expectedException);
+    } catch (RuntimeException actualException) {
+      assertEquals(expectedException.toString(), actualException.toString());
+    }
   }
 
-  @Test(expected = Exception.class)
+  @Test
   public void testValueFeatureWithWhitespaceValue() throws Exception {
-    loadFeature("c2", ValueFeature.class.getCanonicalName(),
-        "{\"value\":\" \"}");
+    final RuntimeException expectedException = 
+        new RuntimeException("mismatch: '0'!='400' @ responseHeader/status");
```

Comment:
2 things. 1) I don't know if this exists, but the error should reflect that you called solr and had some failure. Aka should return (and throw) a "CallingSolrGotAnError" exception. 2) The error message should be more explicit. It should mention what you tried to do (create a feature with white space), what you expected, and what happened instead.